.releaserc

```diff
@@ -8,9 +8,30 @@
   ],
   "plugins": [
     [
-      "@semantic-release/commit-analyzer", {
+      "@codedependant/semantic-release-docker",
```

Comment:
Move this plugin right before the backmerge plugin. It should publish the container after the release is available on GitHub.

Reply:
I didnt take the order into the consideration, thanks.